tvm/version_manager/infrastructure/version_manager_git_repository.py

```diff
@@ -0,0 +1,49 @@
+import json
+import os
+import pathlib
+from typing import List
+
+import requests
+from tvm.version_manager.domain.tutor_version import TutorVersion
+from tvm.version_manager.domain.version_manager_repository import (
+    VersionManagerRepository,
+)
+
+
+class VersionManagerGitRepository(VersionManagerRepository):
+    VERSIONS_URL = "https://api.github.com/repos/overhangio/tutor/tags"
+    TVM_PATH = pathlib.Path.home() / ".tvm"
+
+    def get_version_from_api(self, limit: int = 10):
+        api_info = requests.get(f"{self.VERSIONS_URL}?per_page={limit}").json()
+        return api_info
+
+    def list_versions(self) -> List[TutorVersion]:
+        api_info = self.get_version_from_api()
+        return [TutorVersion(x.get("name")) for x in api_info]
+
+    def get_tutor_local_versions(self):
+        """Return a list of strings with the local version installed. If None, returns empty array."""
+        if os.path.exists(f"{self.TVM_PATH}"):
+            return [
+                x
+                for x in os.listdir(f"{self.TVM_PATH}")
+                if os.path.isdir(f"{self.TVM_PATH}/{x}")
+            ]
+        return []
+
+    def get_active_tutor_version(self) -> str:
```

Comment:
We should convert this method to a static method because we are only using 'self' to get the "TVM_PATH", and we can replace it with an argument. 

```python
@staticmethod
def current_version(tvm_path: str) -> Optional[TutorVersion]:
    ....
```

Reply:
Don't forget to add this new static method as an abstracmethod in VersionManagerRepository.

```python
@staticmethod
@abstractmethod
def ... ():
```